src/ngraph/runtime/cpu/cpu_cse.cpp

```diff
@@ -29,7 +29,26 @@ using namespace std;
 
 static bool cse_convertlayout(std::shared_ptr<Node> a, std::shared_ptr<Node> b)
 {
-    return false;
+    NGRAPH_DEBUG << "In cse_convertlayout for " << a->get_name() << " and " << b->get_name();
+
+    auto ar_a = std::static_pointer_cast<runtime::cpu::op::ConvertLayout>(a);
+    auto ar_b = std::static_pointer_cast<runtime::cpu::op::ConvertLayout>(b);
+
+    // gets the tensor layout from the given node
+    auto get_tl = [](std::shared_ptr<Node> node) {
```

Comment:
if you call this lambda `get_tensor_layout` you won't probably need that comment :-)

Reply:
@Krovatkin https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/cpu/op/convert_layout.cpp#L63